README.md

```diff
@@ -113,19 +113,64 @@ There is a provided docker-compose file (docker-compose-confluent-cloud.yml) tha
 ## Note
 This has only been tested with Confluent Cloud but technically all SASL authenticated Kafka brokers can be reached using this method.
 
-# Run Script
-The run.sh script can be utilized to run the PPM manually.
+# Running the Project
+## Run Script
+The `run.sh` & `run_with_endpoint.sh` scripts can be utilized to run the S3D manually.
```

Comment:
Let's beef this documentation up a bit. From this, we don't know what run and run_with_endpoint are or how they differ.

Reply:
Additional descriptions of the scripts have been added.